docs/source/docker/docker.rst

```diff
@@ -0,0 +1,86 @@
+==================================================
+NI Linux Real-Time and Docker
+==================================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local: 
+
+Introduction
+============
+
+The `Docker platform`_ allows for developers to run applications in consistent environments across
+hardware and operating systems.
+Docker can be used on NI Linux Real-Time with NI hardware for test and measurement use cases.
+The document encompasses the basics of how to install Docker and run containers from an
+NI Linux Real-Time system.
+
+.. _Docker platform: https://www.docker.com
+
+Installation
+============
+
+To install Docker on an NI Linux Real-Time system, simply run the following command
+from your target:
+
+.. code:: bash
+    
+    opkg install docker-ce
+
+NOTE: Currently Docker will not function correctly if you have LabVIEW RT installed
```

Comment:
Can use the ["note" admonition][1] for this.

```suggestion
.. note::

    Currently Docker will not function correctly if you have LabVIEW RT installed.
```

[1]: <https://docutils.sourceforge.io/docs/ref/rst/directives.html#specific-admonitions>